source/fundamentals/indexes.txt

```diff
@@ -144,13 +150,47 @@ created above.
    :language: js
    :start-after: begin-query
    :end-before: end-query
+   :dedent:
 
 Multikey indexes behave differently from non-multikey indexes in terms of
 query coverage, index bound computation, and sort behavior. For a full
 explanation of multikey indexes, including a discussion of their behavior
 and limitations, refer to the :manual:`Multikey Indexes page
 </core/index-multikey>` in the MongoDB manual.
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations. When you create a clustered
+index, you also create a :v5.3:`clustered collection
```

Comment:
Issue: Sentence structure implies that the index creates the collection, instead of the command to create the collection including an index parameter
Suggestion: "You must specify a clustered index to create a clustered collection."